src/jit/MemoryInl.h

```diff
@@ -29,6 +29,7 @@ struct MemAddress {
 #endif /* SLJIT_32BIT_ARCHITECTURE */
 #if defined(ENABLE_EXTENDED_FEATURES)
         CheckNaturalAlignment = 1 << 5,
+        AbsoluteAddress = 1 << 6,
```

Comment:
Is `AbsoluteAddress` used only for extended features?
Would you please briefly introduce the role of this enum value ?

Reply:
I will add a comment. This option limits the result addressing form to a single base register with on offset.

Reply:
> This option limits the result addressing form to a single base register with on offset.

May I ask you more about this?
That is, why is this features applicable only for atomic operations?
I think that there are other possible cases that result addresses could be presented in a single register.

Reply:
Let me give you a background:

Normally, memory access can use all addressing modes of sljit such as:
[base_reg+offset], [absolute_offset], [base_reg+(offset_reg << shift)]

Hence, the address checker can return any of them without limitations.

However, (atomic) callbacks and atomic operations can only use [base_reg] addressing mode. This flag forces to produce only this type of addressing mode.

Since [base_reg] is a subset of [base_reg+offset] with offset 0, sometimes this addressing mode is returned without setting the flag.

Reply:
Thanks for your kind explanation.
I'm not familiar with sljit system, so I need to learn about JITC as well.